scenarios/kubernetes_e2e.py

```diff
@@ -439,6 +440,13 @@ def read_gcs_path(gcs_path):
     print >>sys.stderr, "Read GCS Path: %s" % loc
     return loc
 
+def get_gcs_path(args):
```

Comment:
you only need `args.use_shared_build` and `args.gcs_shared` here, don't have to pass in all the args.

and please add a function docstring

Reply:
agreed, the method above needs a docstring too.